### PR TITLE
Restore pattern to working condition

### PR DESCRIPTION
**Pull request by moficodes**
1. Removed one-click-deploy to IBM Cloud. Until that is fixed should not have that option available for users.
2. Fixed port mismatch that was causing failure in running
3. Added more documentation and images for making IBM Cloud private deployment easier to follow
4. Changed all instance of bx to ibmcloud. 
5. Added troubleshooting section for common issues with cli and icp.

### CODE REVIEW

**Comment by stevemart** [comment]
Thanks for the PR @moficodes !

The changes look good to me. @ykoyfman want to review this change?

I don't mind the one-click deploy being removed. But it seems like the Travis job is now broken hard. We can merge this we should fix that, too. A quick glance seems like we'll need to bump the node version this is using, and migrate a few commands from `bx` -> `ibmcloud` and `idt` to `ibmcloud`

File: README.md [hunk 1]
```diff
@@ -462,14 +442,21 @@ From within the support folder, edit your .env to include your newly acquired cr
...
+2. IBM Cloud Private common errors and solutions can be found [here](https://www.ibm.com/support/knowledgecenter/en/SS8G7U_18.2.0/com.ibm.app.mgmt.doc/content/trouble_common_deployment_errors.htm)
+3. IBM Cloud Private on local vm not working? Check ICP version, Vagrant version and VirtualBox version carefully. Update them if necessary. If all else fails, `vagrant destroy` and `vagrant up` again to reset the cluster.
+ 
+
 # Learn more
 
 * **Artificial Intelligence Code Patterns**: Enjoyed this Code Pattern? Check out our other [AI Code Patterns](https://developer.ibm.com/code/technologies/artificial-intelligence/).
 * **Data Analytics Code Patterns**: Enjoyed this Code Pattern? Check out our other [Data Analytics Code Patterns](https://developer.ibm.com/code/technologies/data-science/)
-* **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde) with all of our Code Pattern videos
+* **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewOibmcloud5N7uGZ5FKH0Fde) with all of our Code Pattern videos
```
Comment (stevemart) [inline README.md]: looks like `bx` was swapped in this URL :)

can you put the original link back: https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde

File: README.md [hunk 2]
```diff
@@ -462,14 +442,21 @@ From within the support folder, edit your .env to include your newly acquired cr
 Redeploy the support microservice, the support feature should now be accessible through the portal.
 
 ```
-$ bx dev deploy
+$ ibmcloud dev deploy
 ```
 
+# Troubleshooting
+
+1. Trouble with IBM Clouc Cli? [Check Here](https://console.bluemix.net/docs/cli/ts_createapps.html#troubleshoot)
```
Comment (stevemart) [inline README.md]: typo with `Clouc`

**Review by stevemar — COMMENTED** [review]
i'm OK with removing the one-click deploy.

@moficodes can you make the changes requested

File: README.md (hunk 2 above)
Comment (ykoyfman) [reply README.md]: Cli -> CLI

**Comment by ykoyfman** [comment]
Mofi, thanks for fixing this pattern. We should merge this but also clean up a lot of minor inconsistencies in the text (vm/VM cli/CLI).

File: README.md (hunk 2 above)
Comment (moficodes) [reply README.md]: Fixed the typos listed. 
I did another scan of the document. Did not spot another one.

**Review by stevemar — APPROVED** [review]
Looks good to me. The master branch is fixed, so we can neve merge this

**Pull request merged**